examples/asr/transcribe_speech.py

```diff
@@ -48,10 +48,10 @@
 Transcribe audio file on a single CPU/GPU. Useful for transcription of moderate amounts of audio data.
 
 # Arguments
-  model_path: path to .nemo ASR checkpoint
+  nemo_model_file: path to .nemo ASR checkpoint
   pretrained_name: name of pretrained ASR model (from NGC registry)
   audio_dir: path to directory with audio files
-  dataset_manifest: path to dataset JSON manifest file (in NeMo format)
+  input_manifest: path to dataset JSON manifest file (in NeMo format)
```

Comment:
Agreed, these name changes are just going to break workflows for other teams. Please revert

Reply:
I'm always crying when I see  **--dataset_manifest** and  **--output_filename**, but I'm not sure if it is a really good reason

Reply:
Haha I agree about them being bad names, but I think I'll take a bad name in one place in order to not break many workflows.

Reply:
Or what we can do - keep the old names as None values, and also add the new arguments. If user provides either, good enough we populate the one with None with the correct value, otherwise we raise an error if user provides both new and old value

Reply:
Won't that effectively double the amount of flags? It works for compatibility, but that's gonna be an eyesore.